src/System.Collections/tests/Generic/SortedSet/SortedSet.Generic.Tests.cs

```diff
@@ -60,22 +60,37 @@ public void SortedSet_Generic_Constructor_IEnumerable_Null()
         public void SortedSet_Generic_Constructor_IEnumerable_IComparer(EnumerableType enumerableType, int setLength, int enumerableLength, int numberOfMatchingElements, int numberOfDuplicateElements)
         {
             IEnumerable<T> enumerable = CreateEnumerable(enumerableType, null, enumerableLength, 0, 0);
+#if netcoreapp
```

Comment:
I think we need to use the ifdef's only if we cannot refactor the code in the original file, otherwise we should split it to other file. I am not seeing the decision has need to be taken according how many lines we need to split to other files.